lib/clAdapters/clMockAdapter.ts

```diff
@@ -27,88 +41,81 @@ class CLMockAdapter extends BaseAdapter {
       }
     });
   }
-};
-
-
 
-export default CLMockAdapter;
+  init() {
+    const p = new Promise((resolve, reject) => {
+      if (this.credentials.appId === '123456' && this.credentials.apiKey === '999999') {
+        resolve(this);
+      } else {
+        reject('Wrong credentials, please use appId "123456" and apiKey "999999"');
+      }
+    });
 
+    return p;
+  }
 
-/**
- * @override
- */
-CLMockAdapter.prototype.init = function() {
-  const _this = this;
-  const p = new Promise((resolve, reject) => {
-    if (_this.credentials.appId === '123456' && _this.credentials.apiKey === '999999') {
-      resolve(_this);
-    } else {
-      reject('Wrong credentials, please use appId "123456" and apiKey "999999"');
+  /**
+   * Reflects given field in this `result` format:
+   * ```
+   * {
+   *   {fieldType}ExtId: rand # between 1-50,
+   *   {extId}: Float (0-1.0)
+   * }
+   * ```
+   *
+   * For example, for a "Fields.Types.USER" field w/extId 'user#utilizationRate', would return:
+   * ```
+   * {
+   *   userExtId: 50,
+   *   'user#utilizationRate': 0.55
+   * }
+   * ```
+   *
+   * Returns between 0-50 results in pages of `query.limit` (default 5)
+   * @override
+   */
+  getFieldData(field: Fields.Field, query: any) {
+    query = query || {};
+    let typeName: string;
+    console.log(field);
```

Comment:
That was prior implementation for this mock adapter that should be left. The main change was moving that method implementation from a prototype method into the new class definition (looks like TypeScript compiler can't tell if you implemented required interface methods if they are declared on prototype).